### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by nomusictuesdays**
Fixed date to show correct month and add leading zero to months 01-09

### CODE REVIEW

File: apps/bradbury/app.js [hunk 1]
```diff
@@ -35,9 +35,9 @@ function draw() {
   g.setFont("5x9Numeric7Seg",1);
   g.setFontAlign(1,1);
   g.setColor(0,0,1);
-  g.drawString(yyyy+" "+("0"+mm)+" "+dd, 100, 65, 0);
+  g.drawString(yyyy+" "+("0"+mm)+" "+("0"+dd), 100, 65, 0);
```
[the remark this answers is not in the document]
Comment (nomusictuesdays) [reply apps/bradbury/app.js:38]: Thank you, yes I realized it the next day when my date said 2022-05-010. Embarrasing. But thanks for the tip, I'll implement that asap.

**Comment by gfwilliams** [comment]
Any chance of getting this fixed, or shall I close this PR?  it's been open for 4 months now. (and please could you bump the version number and add a ChangeLog entry?)

**Comment by nomusictuesdays** [comment]
> Any chance of getting this fixed, or shall I close this PR? it's been open for 4 months now. (and please could you bump the version number and add a ChangeLog entry?)

Go ahead and close this PR, it's been a few months since I've had time to look at this, I can't even remember where I was on this project. And as a noob to github, I'd need to go over the instructions again and reorient myself to the system. Thanks!

**Comment by gfwilliams** [comment]
Ok, thanks!

**Pull request closed without merging**